docs/store/selectors.md

```diff
@@ -27,6 +27,53 @@ export const selectFeature = (state: AppState) => state.feature;
 export const selectFeatureCount = createSelector(selectFeature, (state: FeatureState) => state.counter);
 ```
 
+
+### Example with multiple state slices
+
+The `createSelector` can be used to select some data from the state based on several slices of the same state.
+
+For example, imagine you have a `selectedUser` object in the state. You also have an `allBooks` array of book objects.
+
+And you want to be able to show only the books that belong to your `selectedUser` if there is one. Otherwise you want to show them all. 
+
+You can use the `createSelector` to achieve just that. Your visible books will always be up to date even if you update them in `allBooks` and they will always show the books that belong to your user if there is one selected, and will show all the books when there is no user selected.
+
+It will behave like if you had another portion of the state that is magically synchronized with the rest of your state. But it is actually just some of your state filtered by another section of the state.
+
+```ts
+// reducers.ts
+import { createSelector } from '@ngrx/store';
+
+export interface User {
+  id: number;
+  name: string;
+}
+
+export interface Book {
+  id: number;
+  userId: number;
+  name: string;
+}
+
+export interface AppState {
+  selectedUser: User;
+  allBooks: Book[];
+}
+
+export const selectUser = (state: AppState) => state.selectedUser;
+export const selectAllBooks = (state: AppState) => state.allBooks;
+
+export const selectVisibleBooks = createSelector(selectUser, selectAllBooks, (selectedUser: User, allBooks: Books[]) => {
+  if (selectedUser && allBooks) {
+    return allBooks.filter((book: Book) => book.userId === selectedUser.id);
+  } else {
+    return allBooks;
+  }
+});
+```
+
+You can imagine how this could be extended to create complex state selections, based on several parts of your state. Actually, you can use `createSelector` with up to 8 selector functions (the example above uses just 2 selector functions).
```

Comment:
"The `createSelector` function can take up to 8 selector function for more complete state selections" sounds clearer to me. Move this section after the first sentence in this section